examples/GANs/3D-IWGAN/train.py

```diff
@@ -14,55 +14,41 @@
 
 import argparse
 import json
-import numpy as np
 import os
 import torch
-from torch.autograd import Variable
 import torch.optim as optim
 from torch.utils.data import DataLoader
-import sys
 from tqdm import tqdm
 
 
-from utils import gradient_penalty, calculate_gradient_penalty
+from utils import calculate_gradient_penalty
 from architectures import Generator, Discriminator
 
 import kaolin as kal 
-"""
-Commandline arguments
-"""
-parser = argparse.ArgumentParser()
-parser.add_argument('-expid', type=str, default='GAN', help='Unique experiment identifier.')
-parser.add_argument('-device', type=str, default='cuda', help='Device to use')
-parser.add_argument('-categories', type=str,nargs='+', default=['chair'], help='list of object classes to use')
-parser.add_argument('-epochs', type=int, default=50000, help='Number of train epochs.')
-parser.add_argument('-batchsize', type=int, default=50, help='Batch size.')
-parser.add_argument('-val-every', type=int, default=5, help='Validation frequency (epochs).')
-parser.add_argument('-print-every', type=int, default=2, help='Print frequency (batches).')
-parser.add_argument('-logdir', type=str, default='log', help='Directory to log data to.')
-parser.add_argument('-save-model', action='store_true', help='Saves the model and a snapshot \
-	of the optimizer state.')
-args = parser.parse_args()
 
 
-cats = ['airplane','bathtub','bed','bench','bookshelf','bottle','bowl','car','chair','cone','cup',
-'curtain','desk','door','dresser','flower_pot','glass_box','guitar','keyboard','lamp',
-'laptop','mantel','monitor','night_stand','person','piano','plant','radio','range_hood',
-'sink','sofa','stairs','stool','table','tent','toilet','tv_stand','vase','wardrobe','xbox']   
-"""
-Dataset
-"""
-train_set = kal.datasets.ModelNet(root ='../../datasets/',categories = args.categories, \
-			single_view= True, download = True)
-dataloader_train = DataLoader(train_set, batch_size=args.batchsize, shuffle=True, 
-	num_workers=8)
+parser = argparse.ArgumentParser()
+parser.add_argument('--modelnet-root', type=str, help='Root directory of the ModelNet dataset.')
+parser.add_argument('--cache-dir', type=str, default=None, help='Path to write intermediate representation to.')
+parser.add_argument('--expid', type=str, default='GAN', help='Unique experiment identifier.')
```

Comment:
Is this really  "Unique" if it have the same default id than in https://github.com/NVIDIAGameWorks/kaolin/blob/master/examples/GANs/3DGAN/train.py#L32 ?

Reply:
Fixed.